File: 백준/Bronze/10813. 공 바꾸기/공 바꾸기.cc
#include <bits/stdc++.h>
#define FastIO ios_base::sync_with_stdio(false); cin.tie(nullptr); cout.tie(nullptr);
using namespace std;

int main() {
    FastIO

    int N, M;
    cin >> N >> M;

    vector<int> save(N + 1);

    for (int i = 1; i <= N; i++) {
        save[i] = i;
    }

    for (int i = 0; i < M; i++) {
        int a, b;
        cin >> a >> b;

        int temp = save[b];
        save[b] = save[a];
        save[a] = temp;
    }

    for (int i = 1; i <= N; i++) {
        cout << save[i] << ' ';
    }

    return 0;
}

File: 백준/Bronze/10813. 공 바꾸기/공 바꾸기.py
N, M = map(int, input().split())

save = []

for i in range(1, N + 1):
    save.append(i)

for i in range(0, M):
    a, b = map(int, input().split())

    temp = save[a - 1]
    save[a - 1] = save[b - 1]
    save[b - 1] = temp

for i in range(0, N):
    print(save[i], end=' ')